.github/workflows/test.yml

```
@@ -230,6 +230,126 @@ jobs:
         # which pixi should be absolute
         which pixi | grep -q "^/"
 
+  existing-pixi-bin:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        ignoreReason: [none, version, version-latest, url, bin-path]
```

```suggestion
        ignore-reason: [none, version, version-latest, url, bin-path]
```

we use kebab-case in GHA

---

fixed